docs/multi-manager-setup.md

```diff
@@ -0,0 +1,157 @@
+# High availability in Swarm
+
+In Docker Swarm, the **swarm manager** is responsible for the entire cluster and manages the resources of multiple `docker` hosts at scale. If the swarm manager dies, you must create a new one and deal with an interruption of service.
+
+The *High Availability* feature of Docker Swarm allows a swarm to handle gracefully the failover of a manager instance. Using this feature, you can create a single **primary manager** instance and multiple **replica** instances.
+
+A primary manager is the main point of contact with the Docker Swarm cluster. You can create and talk to replica instances. Requests issued on a replica are automatically proxied to the primary manager. If the primary manager fails, a replica takes away the lead. In this way, you always keep a point of contact with the cluster.
+
+## Setup primary and Replicas
+
+This section explains how to set up Docker Swarm using multiple **swarm managers**.
+
+### Assumptions
+
+You need either a `Consul`, `etcd` or `Zookeeper` cluster. This procedure is written assumes a `Consul` server running on address `192.168.42.10:8500`.  The sample swarm configuration has three machines:
```

Comment:
"This procedure is written assumes a.." looks incorrect, I think you can remove "is written". Or, if you want to keep the "is written" part;

```Markdown
This procedure is written, assuming a `Consul` server running on address `192.168.42.10:8500`.